scriptworker/poll.py

```diff
@@ -121,15 +127,33 @@ def get_azure_urls(context):
 
     Returns:
         dict: the claimTask json
+
+    Raises:
+        ScriptWorkerException: on failure of request_function
     """
+    # this can raise a ScriptWorkerException on failure
```

Comment:
Yeah, I wanted to point out that that was the one line where I saw a potential uncaught exception.